gems/pending/VMwareWebService/MiqVimEventMonitor.rb

```diff
@@ -2,11 +2,12 @@
 require 'VMwareWebService/MiqVimInventory'
 
 class MiqVimEventMonitor < MiqVimInventory
-  def initialize(server, username, password, eventFilterSpec = nil, pgSize = 100)
+  def initialize(server, username, password, eventFilterSpec = nil, pgSize = 100, maxWait = 60)
```

Comment:
@Fryguy pgSize should be in the config, https://github.com/ManageIQ/manageiq/blob/master/config/vmdb.tmpl.yml#L294
We should be passing both config values in here https://github.com/ManageIQ/manageiq/pull/7335/files/2042b4c5ce452edd75de84ddc127eac4ee02697b#diff-6380f89005d7f8ae26902652dba5fd7bR9